packages/core/src/makeStyles.ts

```diff
@@ -43,10 +46,8 @@ export function makeStyles<Slots extends string | number>(stylesBySlots: StylesB
       }
     }
 
-    if (insertionCache[rendererId] === undefined) {
-      renderer.insertCSSRules(cssRules!);
-      insertionCache[rendererId] = true;
-    }
+    insertStyles(renderer, dir, cssRules!);
```

Comment:
nit: maybe at this point the `computeClasses` function name above was never quite clear. This step does not only compute the classes but also inserts them to DOM